### ISSUE TITLE
grpc calls not evaluated unless they're the last ones in the exec block

### ISSUE THREAD

**Issue by larrywest42** [phiSgr/gatling-grpc#21]
This is almost certainly my misunderstanding, but I can't see how to get Gatling to evaluate the `grpc()` calls I have in

```scala
.exec((session : Session) => {
    grpc("Make the call")
       .rpc(StudentServiceGrpc.getListMethod)
       .payload(Empty.getDefaultInstance())
       .check(...)
       .extract(_.value.some)(_ saveAs "ListResponse")
    })
```

...won't work: 
```
type mismatch;
 found   : com.github.phisgr.gatling.grpc.action.GrpcCallActionBuilder[com.google.protobuf.Empty,com.work.platform.grpc.student.api.definition.Students]
 required: io.gatling.commons.validation.Validation[io.gatling.core.session.Session]
```

...so I have to add a line of `session` right after the `extract`.  But then the `grpc` call is never evaluated, perhaps because Gatling (Scala?) thinks it has no effect (it's just a builder, and nothing refers to it).  But I can't see how to make it be called ... `.build()` takes parameters I have no access to.

FWIW, I wrote this up, with a related issue, on [SO](https://stackoverflow.com/questions/59506792/how-to-pass-gatling-session-attributes-in-an-exec-invoking-another-library-ge), but then thought perhaps I should just ask you about this part.

**Comment by phiSgr** [comment]
Please don’t use the issue board to ask questions. Have you looked at the examples and see how things are different from your code?

This library is a plugin to Gatling, so you will find better explanations on Gatling from them than from me. In fact your error is pointed out in its documentation. 
https://gatling.io/docs/current/general/scenario/

**Comment by phiSgr** [comment]
Asking in terms of implementation details that you do not know about is like asking “How do I pass through wall” when you just want to exit the building. You just get “No, you don’t.”

**Comment by larrywest42** [comment]
I _completely_ understand that you don't want to deal with questions here. This is absolutely fine.

But being condescending in the (second) response strikes me as gratuitous.

Goodbye.

**Comment by phiSgr** [comment]
That remark on how to ask questions has nothing to do with the question being in the issue board. I want to be helpful but what is written here gives me nothing to work with. Even in the SO question, the end goal is not that clear.

In hindsight I should have just linked [How To Ask Questions The Smart Way](http://www.catb.org/~esr/faqs/smart-questions.html#goal), but this analogy popped into my head and I found it very fitting. It's unfortunate that I did not convey the tone in writing and you found it condescending.
